charts/redpanda/templates/certs.yaml

```diff
@@ -23,6 +23,8 @@ limitations under the License.
   {{- $values := .Values }}
   {{- range $name, $data := .Values.tls.certs }}
     {{- $d := $data.duration }}
+    {{- if (empty $data.secretRef ) }}
+
```

Comment:
```suggestion
    {{- if (empty $data.secretRef ) }}
    {{- $d := $data.duration }}

```

Reply:
Another nit, for all `{{-` between here and `{{- end }}`, add an indent. It just makes figuring out the nesting a little bit easier when reading.